<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayUser</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1047</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #1b1f2a, stop:1 #10131a);</string>
  </property>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>580</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Style for the Delete button */
QPushButton#pushButton_3 {
    background-color: #FF1414; /* Muted dark red */
    color: #E0E6ED; /* Light gray text */

}
QPushButton#pushButton_3:hover {
    background-color: #ff5d5d; /* Lighter red on hover */
}
QPushButton#pushButton_3:pressed {
    background-color: #6A3A3A; /* Darker red when pressed */
}
    border: 1px solid white; /* Subtle blue-gray border */
    border-radius: 1px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/img/deleteIcon.png</normaloff>:/img/deleteIcon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_10">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
    border: 1px solid white; /* Subtle blue-gray border */
    border-radius: 15px;
color: rgb(24, 24, 24);
</string>
   </property>
   <property name="placeholderText">
    <string>      Search by user's name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pdfButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>580</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Style for the Modify button */
QPushButton#pdfButton {
	background-color: #2B2F37;
}
QPushButton#pdfButton:hover {
    background-color:#343842 ; /* Darker green on hover */
}
QPushButton#pdfButton:pressed {
    background-color: #1e2127; /* Even darker green when pressed */
}
    border: 1px solid white; /* Subtle blue-gray border */
    border-radius: 1px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/img/pdfIcon.png</normaloff>:/img/pdfIcon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="photoLabel">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>370</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border: 1px solid #3A3D4A;
border-radius: 12px;
box-shadow: 0 4px 10px rgba(0, 0, 0, 0.2);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>701</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Style for the tableWidget */
QTableView {
    background-color: #2D3748; /* Dark background matching the app */
    color: #FFFFFF; /* White text for readability */
    border: 1px solid #4A4E58; /* Subtle border matching button hover color */
    gridline-color: #4A4E58; /* Subtle grid lines */
}

/* Style for table items */
QTableView::item {
    border: 1px solid #4A4E58; /* Subtle cell borders */
    padding: 5px; /* Spacing inside cells */
}

/* Alternating row colors */
QTableView::item:nth-child(even) {
    background-color: #2D3748; /* Darker shade for even rows */
}

QTableView::item:nth-child(odd) {
    background-color: #394252; /* Slightly lighter shade for odd rows */
}

/* Hover effect for rows */
QTableView::item:hover {
    background-color: #4A4E58; /* Lighter shade on hover */
}

/* Style for the header */
QHeaderView::section {
    background-color: #F5A623; /* Accent color for header */
    color: #1A2633; /* Dark text for contrast */
    font-weight: bold; /* Bold header text */
    border: 1px solid #4A4E58; /* Border to match table */
    padding: 5px; /* Spacing inside header */
}

/* Remove selection background */
QTableView::item:selected {
    background-color: #4A4E58; /* Same as hover to avoid clashing */
    color: #FFFFFF; /* Keep text white */
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Style for the Adjust Team title label */
QLabel#label_4 {
    background-color: transparent; /* Dark background matching the app */
    color: #FFFFFF; /* White text for contrast */
    font-size: 25px; /* Larger font for title emphasis */
    font-weight: bold; /* Bold for a strong title appearance */


    padding: 5px 10px; /* Padding for a balanced look */
}</string>
   </property>
   <property name="text">
    <string>Display User</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ModifyUser">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>580</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Style for the Modify button */
QPushButton#ModifyUser {
    background-color:#D8A353 ; /* Green to match the accent */
    color: #1A2633; /* Dark blue text */

}
QPushButton#ModifyUser:hover {
    background-color: #eeb35b; /* Darker green on hover */
}
QPushButton#ModifyUser:pressed {
    background-color:#a37b3f ; /* Even darker green when pressed */
}
    border: 1px solid white; /* Subtle blue-gray border */
    border-radius: 1px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/img/editIcon.png</normaloff>:/img/editIcon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>44</width>
     <height>44</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/img/search.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>320</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Style for the Adjust Team title label */
QLabel#label_5 {
    background-color: transparent; /* Dark background matching the app */
    color: #FFFFFF; /* White text for contrast */
    font-size: 15px; /* Larger font for title emphasis */
    padding: 5px 10px; /* Padding for a balanced look */
}</string>
   </property>
   <property name="text">
    <string>User's Picture</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>741</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(25, 31, 42);
    color: #FFFFFF;
    font-family: 'Segoe UI', Arial, sans-serif;
    font-size: 12px;
    border-radius: 15px;
    padding: 15px;
    spacing: 2px;
</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3"/>
  </widget>
  <zorder>widget</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>lineEdit_10</zorder>
  <zorder>pdfButton</zorder>
  <zorder>photoLabel</zorder>
  <zorder>tableView_3</zorder>
  <zorder>label_4</zorder>
  <zorder>ModifyUser</zorder>
  <zorder>label</zorder>
  <zorder>label_5</zorder>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
